src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilTest.java

```diff
@@ -379,6 +389,64 @@ public void testEscapeAllControlChars() {
                 JavadocUtil.escapeAllControlChars("1\\r2\\n3\\t"), "invalid result");
     }
 
+    @Test
+    public void testGetContentOfInlineCustomTag() throws Exception {
+        final File testFile = new File("src/test/resources/com/"
+                + "puppycrawl/tools/checkstyle/utils/javadocutil/"
+                + "InputJavadocUtilInlineCustomTag.java");
```

Comment:
Why not use `AbstractPathTestSupport#getPath` here?

Reply:
Done

Reply:
Resolved.